src/vs/workbench/browser/parts/auxiliarybar/auxiliaryBarActions.ts

```diff
@@ -136,3 +136,25 @@ MenuRegistry.appendMenuItems([
 		}
 	}
 ]);
+
+registerAction2(class extends SwitchCompositeViewAction {
+	constructor() {
+		super({
+			id: 'workbench.action.previousAuxiliaryBarView',
+			title: localize2('previousAuxiliaryBarView', 'Previous Secondary Bar View'),
```

Comment:
Can we call it `Previous Secondary Side Bar View`, not sure if it was on purpose but we call it `Secondary Side Bar` everywhere and `Secondary Bar` does weird/unclear